packages/office-ui-fabric-react/src/components/Toggle/Toggle.tsx

```diff
@@ -29,8 +31,8 @@ export class Toggle extends React.Component<IToggleProps, IToggleState> {
     this.state = {
       isChecked: !!(props.checked || props.defaultChecked)
     };
-
-    this._id = getId('Toggle');
+    const { id } = getNativeProps(props, buttonProperties) as React.HTMLProps<HTMLButtonElement>;
```

Comment:
couldn't you just say id = props.id? You're not using the result of the filter function.